app/Providers/AppServiceProvider.php

```diff
@@ -33,6 +34,15 @@ public function boot()
 		Blade::directive('endcanany', function ($b) {
 			return "<?php endif; ?>";
 		});
+
+		Validator::extend('same_when', function ($attribute, $value, $parameters, $validator) {
```

Comment:
Ik vind deze naam te verwarrend mbt de bestaande validatieregel `same` https://laravel.com/docs/7.x/validation#available-validation-rules

Het is een ingewikkeld construct, maar misschien is een naam als `when_then_different` beter? Anders mitigeren door een comment te plaatsen bij het gebruik in `ParticipantRequest` om uit te leggen wat de bedoeling is.

Reply:
Mee eens , en done